doc/sphinx/language/gallina-extensions.rst

```diff
@@ -701,6 +693,9 @@ terminating functions.
 Depending on the ``{…}`` annotation, different definition mechanisms are
 used by ``Function``. A more precise description is given below.
 
+.. todo Théo, should the following info on cmdv:: Function s be moved adjacent
+   to the definition of @fixannot?  Maybe create a new subsection in the Gallina chapter?
```

Comment:
No, because these are extensions of the core language.

The Gallina chapter should note that only the `{ struct }` annotation is part of the core language and that `Fixpoint` does not support `wf` and `measure` (indeed, when trying to use them, we get an error message "Well-founded induction requires Program Fixpoint or Function"). And that there use is documented at ...

Reply:
OK.  Can you give me a short description of when the user has to limit themself to non-extension features?  This can go in the introduction to the extensions chapter and briefly repeated where appropriate (such as for `struct`).

Reply:
Never. It's just the approach we want to follow to explain things in the refactored manual: first explain what the kernel understands, then explain the rich language that Coq provides to the user and which is transformed to the core language of the kernel.

Reply:
Hmm, it seems like we should eventually explain that distinction and the reason for organizing the material that way (just for conceptual/pedagogic reasons).

Reply:
Yes, absolutely, but that's not really the task of the current PR.